spec/features/faq_spec.rb

```diff
@@ -10,7 +10,7 @@
     visit "/faq"
 
     click_on faq_item.question_en
-    expect(strip_html_tags(page.body)).to include(faq_item.answer_en.to_plain_text)
```

Comment:
We actually never need to `strip_html_tags` for the Capybara page because Capybara will do it for you so I replaced all usages of this with just `page`. You will have to do it on the expectation side but it seems to be expensive as many tests will snag on this so avoid if possible